modules/mono/editor/GodotTools/GodotTools/GodotTools.csproj

```diff
@@ -28,6 +28,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="JetBrains.Annotations" Version="2019.1.3.0" ExcludeAssets="runtime" PrivateAssets="all" />
+    <PackageReference Include="JetBrains.Rider.PathLocator" Version="1.0.0" />
```

Comment:
We are getting this warning on restoring this package:

```
warning NU1701: Package 'JetBrains.Rider.PathLocator 1.0.0' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8, .NETFramework,Version=v4.8.1' instead of the project target framework 'net6.0'. This package may not be fully compatible with your project.
```

It seems the package's TFM is _.NET Framework_. If you need the library to support consumers that target both _.NET Framework_ and _.NET Core_, consider changing the package's TFM to `netstandard2.0` which would make it compatible with both. Otherwise, you can also use multi-targeting.